release/create_candidate_pr.py

```diff
@@ -0,0 +1,79 @@
+# pylint: disable=redefined-outer-name, invalid-name
+
+"""
+Creates a new release candidate branch and makes a pull request
+for it into the 'release' branch.
+"""
+
+from __future__ import print_function
+
+import argparse
+
+from release.get_token import get_token
+from release.github_api import GithubApi, RequestFailed
+from release import utils
+
+
+def _build_parser():
+    """ Creates an option parser for this command """
```

Comment:
Pylint as we have it configured doesn't complain about the contents of docstrings in any way.  Pylint 1.5.0 has some experimental stuff, but only about whether you mentioned all the arguments.  I don't know of a thing that would be strict about the formatting of docstrings.